### ISSUE TITLE
Bug in Index/MultiIndex name when reading in Python

### ISSUE THREAD

**Issue by tebeka** [v3io/frames#45]
@yaronha said
>  u need to set the `Index` (column) name not just value,

**Comment by tebeka** [comment]
@yaronha I don't follow, this seems to be working as expected (code same as in [the client](https://github.com/v3io/frames/blob/master/clients/py/v3io_frames.py#L312))

```python
In [9]: indices = [pd.Series([1,2,3], name='index-a'), pd.Series(['x', 'y', 'z'], name='index-b'
   ...: )]                                                                                      

In [10]: mi = pd.MultiIndex.from_arrays(indices)                                                

In [11]: mi                                                                                     
Out[11]: 
MultiIndex(levels=[[1, 2, 3], ['x', 'y', 'z']],
           labels=[[0, 1, 2], [0, 1, 2]],
           names=['index-a', 'index-b'])

In [12]: mi.names                                                                               
Out[12]: FrozenList(['index-a', 'index-b'])

In [20]: df = pd.DataFrame({'x': [10, 20, 30]})                                                 

In [21]: df.index = indices[0]                                                                  

In [22]: df.index                                                                               
Out[22]: Int64Index([1, 2, 3], dtype='int64', name='index-a')
```

**Comment by tebeka** [comment]
This was a bug in handling slice columns which didn't copy the name from the message. Fixed in 2acad9d
